app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,7 @@
+class FoodsController < ApplicationController
+    def search
+        @keyword = params[:q]
+        @results = FoodFacade.search_results(@keyword)
+        @foods = FoodFacade.search(@keyword)
+    end
```

Comment:
Lines 4 and 5 are going to make api calls to the same exact endpoint. Can you think of a way to only make the api call once and gather all the info you need for your view from that one api call?